presto-analyzer/src/main/java/com/facebook/presto/sql/analyzer/Scope.java

```diff
@@ -99,6 +102,11 @@ public RelationType getRelationType()
         return relation;
     }
 
+    public boolean isRequireSingleRowValues()
```

Comment:
I think this could be renamed to `requireSingleRowValues`

Reply:
Thanks, fixed!